src/commands/sbt_cached.yml

```diff
@@ -48,9 +52,9 @@ steps:
               find $HOME/.ivy2 -name "ivydata-*.properties" | xargs rm | true
 
         - save_cache:
-            key: sbt-cache-{{ .Branch }}-{{ checksum "build.sbt" }}-{{ .Environment.CIRCLE_SHA1 }}
+            key: << parameters.cache_prefix >>-{{ .Branch }}-{{ checksum "build.sbt" }}-{{ .Environment.CIRCLE_SHA1 }}
             paths:
-              - "~/.ivy2/cache"
+              - "~/.ivy2"
```

Comment:
This is to save the `publishLocal` artifacts in the cache?

Reply:
to keep `local` as well yes.